blocks-config/image-gallery/class-spectra-image-gallery.php

```diff
@@ -1308,7 +1321,7 @@ private function render_masonry_hover_handler( $mediaArray, $atts ) {
 		private function render_single_media( $mediaArray, $atts ) {
 			// Check if this is part of the Tiled Layout, and if so then check if the current image is focused or not.
 			$focusedClass = '';
-			if ( 'tiled' === $atts['feedLayout'] && ( array_key_exists( $mediaArray['id'], $atts['focusList'] ) && ( true === $atts['focusList'][ $mediaArray['id'] ] ) ) ) {
+			if ( 'tiled' === $atts['feedLayout'] && ( array_key_exists( $mediaArray['id'], $atts['focusListObject'] ) && ( true === $atts['focusListObject'][ $mediaArray['id'] ] ) ) ) {
```

Comment:
```suggestion
			if ( 'tiled' === $atts['feedLayout'] && ! empty( $atts['focusListObject'][ $mediaArray['id'] ] ) ) {
```

please check syntax once

Reply:
Done, replaced with Not Empty check for the current ID's value in the Focus List Object.